apps/commudle-admin/src/app/feature-modules/community-builds/components/community-builds/community-build-card/community-build-card.component.html

```diff
@@ -16,10 +16,10 @@
         <app-badge
           *ngIf="communityBuild.hackathon_team"
           [text]="'Hackathon'"
-          [fontColor]="'com-text-yellow-700'"
-          [fontSize]="'small'"
+          [fontColor]="'com-text-tChocolate-Traditional'"
+          [fontSize]="'mini'"
```

Comment:
let's make mini to extra-small or xs

Reply:
extra small is already made with 10px font size, its 8px font size

Reply:
xxs should be the double extra small